test/stdlib/TestDecimal.swift

```diff
@@ -237,6 +237,10 @@ class TestDecimal : TestDecimalSuper {
                 expectEqual(Decimal(i*j), Decimal(i) * Decimal(j), "\(Decimal(i*j)) == \(i) * \(j)")
                 expectEqual(Decimal(i+j), Decimal(i) + Decimal(j), "\(Decimal(i+j)) == \(i)+\(j)")
                 expectEqual(Decimal(i-j), Decimal(i) - Decimal(j), "\(Decimal(i-j)) == \(i)-\(j)")
+                //Strideable methods
```

Comment:
Nit: please follow the prevailing comment style in this file by putting a space between `//` and `Strideable`.